packages/web3-eth-accounts/src/account.ts

```diff
@@ -625,11 +623,8 @@ export const privateKeyToAccount = (privateKey: Bytes, ignoreLength?: boolean):
 		},
 		sign: (data: Record<string, unknown> | string) =>
 			sign(typeof data === 'string' ? data : JSON.stringify(data), privateKeyBuffer),
-		encrypt: async (password: string, options?: Record<string, unknown>) => {
-			const data = await encrypt(privateKeyBuffer, password, options);
-
-			return JSON.stringify(data);
-		},
+		encrypt: async (password: string, options?: Record<string, unknown>) =>
+			encrypt(privateKeyBuffer, password, options),
```

Comment:
```suggestion
			await encrypt(privateKeyBuffer, password, options),
```
should be using `await`?

Reply:
No. It is unnecessary because we do not do anything with the result and do not catch an error.